src/collection-browser.ts

```diff
@@ -644,12 +646,19 @@ export class CollectionBrowser
   private get manageBarTemplate(): TemplateResult {
     return html`
       <manage-bar
-        label=${this.manageViewLabel}
-        showSelectAll
-        showUnselectAll
+        .label=${this.manageViewLabel}
+        ?showSelectAll=${true}
+        ?showUnselectAll=${true}
+        ?enableRemoveButton=${this.manageViewRemoveButtonEnable}
         @removeItems=${this.handleRemoveItems}
-        @selectAll=${() => this.dataSource.checkAllTiles()}
-        @unselectAll=${() => this.dataSource.uncheckAllTiles()}
+        @selectAll=${() => {
```

Comment:
Per comment below: if you use a method rather than a state for `enableRemoveButton`, this change is unnecessary.